UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/IRnwRepository.cs

```diff
@@ -0,0 +1,11 @@
+﻿using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO;
+
+namespace UKHO.MaritimeSafetyInformation.Web.Services
+{
+    public interface IRnwRepository 
+    {
+        Task<bool> AddRadioNavigationWarnings(RadioNavigationalWarnings radioNavigationalWarnings, string correlationId);
+
+        List<WarningType> GetWarningType();
```

Comment:
Refer changes made to GetWarningTypes() in branch "Feature/46539 list all rnw records" to keep it consistent.

Reply:
done required changes